elevate-cpanel

```diff
@@ -4388,6 +4388,7 @@ EOS
         'kernelcare',
         'updates',
         qr/^wp-toolkit-(?:cpanel|thirdparties)$/,
+        'zabbix',
```

Comment:
you're modifying a generated file here. The second you merge this, it'll get clobbered.